src/AppBundle/Entity/LegislativeCandidate.php

```diff
@@ -246,4 +256,14 @@ public function hasWebPages(): bool
     {
         return $this->websiteUrl || $this->twitterPageUrl || $this->facebookPageUrl || $this->donationPageUrl;
     }
+
+    public function getCareer(): string
```

Comment:
`string` is not true since you don't have a construct to initialize the value.
https://3v4l.org/5TSR9

It should be `?string`